packages/components/organisms/f-checkout/src/index.js

```diff
@@ -8,7 +8,7 @@
 // Import vue component
 import VueCheckout from '@/components/Checkout.vue';
 import CheckoutModule from '@/store/checkout.module';
-import AnalyticsModule from '@/store/analytics.module';
+import CheckoutAnalyticsModule from '@/store/checkoutAnalytics.module';
```

Comment:
Not sure this one needs to change? When you register the module you can attach the name space there which should avoid the clash?

Reply:
Hey, I can't remember if we talked since this, but it's so the `...mapState('analytics` ....` calls don't get confused with the CoreWeb analytics module.